src/functions/newsletter-submission.js

```diff
@@ -0,0 +1,51 @@
+const fetch = require("node-fetch");
+const { BUTTONDOWN_API_KEY } = process.env;
```

Comment:
👍 Nice. Good thinking putting this in an env var. You should be a note in the README about this. Where is this env var set? Is it set via the Netlify UI? Somewhere else? We have an existing Buttondown section where it can go.

Reply:
Updated.